Ohana/Classes/Common/DataProviders/OHCNContactsDataProvider.m

```diff
@@ -82,6 +85,16 @@ + (NSString *)providerIdentifier
 
 #pragma mark - Private
 
+- (void)triggerUserAuthentication
+{
+    CNContactStore *contactStore = [[CNContactStore alloc] init];
```

Comment:
this is happening within the OHCNContactsDataProvider. It isn't the provider's responsibility to check if the tech it is built around is available evert time it tries to use it. (And it doesn't do so in the myriad of other places it uses CNContacts).